src/renderer/components/Mediator/index.vue

```diff
@@ -1,25 +1,74 @@
 <template>
   <el-row>
     <div style="margin-bottom: 1em;">
-        <p>Mediator Routes</p>
-        <el-collapse v-model="expanded_items">
-          <ul class="list">
-            <el-collapse-item
-              v-for="r in mediator_routes"
-              v-bind:title="route_name(r)"
-              :name="r.recipient_key"
-              :key="r.connection_id">
-              <el-row :key="r.connection_id">
-                <div>
-                  <vue-json-pretty
-                    :deep=2
-                    :data="r">
-                  </vue-json-pretty>
-                </div>
-              </el-row>
-            </el-collapse-item>
-          </ul>
-        </el-collapse>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a class="navbar-brand" href="#">Mediation Clients</a>
+        <el-button
+          type="primary"
+          icon="el-icon-refresh"
+          @click="fetch_mediator_clients"></el-button>
+      </nav>
+      <el-collapse v-model="expanded_items">
+        <ul class="list">
+          <el-collapse-item
+            v-for="client in mediator_clients"
```

Comment:
this is the first time I have seen the term `mediator_clients` should it be something like `mediator_recipient` to better conform to aries terminology?

Reply:
Mediator recipient might be confused with `recipient keys` so between those two options, I would side with `mediator_clients` still. Open to better names though.